src/App.js

```diff
@@ -112,7 +113,9 @@ function App() {
       {globalData &&
       Object.keys(globalData).length > 0 &&
       globalChartData &&
-      Object.keys(globalChartData).length > 0 ? (
+      Object.keys(globalChartData).length > 0 &&
+      tokensLists &&
+      tokensLists.length > 0 ? (
```

Comment:
We are removing this check in App.js. So move this to TokenList page loader

Reply:
Maybe It's better if I fix this after we merge your PR about the loading changes?

Reply:
I thought about it, and maybe the whole app should wait for the tokens lists to be fetched since the token logos are used almost everywhere, what do you think? The http calls to fetch the tokens are all pretty fast

Reply:
I think most of links in the URL array is getting 404. I will give a check today evening and see. If its fast we can wait till it loads.

Reply:
The 404s were caused by searching the images on trustwallet, but I removed that since it was redundant, all the icons are included in the pre-loaded lists